source/deployment-paradigms/multi-region.txt

```diff
@@ -0,0 +1,257 @@
+.. _arch-center-paradigms-multi-region:
+
+================================
+Multi-Region Deployment Paradigm
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: onecol
+
+Multi-region {+service+} deployments set up {+cluster+} nodes across
+multiple regions (as defined by the cloud providers). Multi-region
+deployments enhance protection in the case of a :ref:`regional outage
+<arch-center-regional-outage>` by automatically rerouting traffic to a
+node in another region for continuous availability and a
+smooth user experience. Multi-region deployments can also enhance
+performance and can help meet compliance requirements for :ref:`data sovereignty
+<arch-center-compliance-data-sovereignty>`, such as the EU's General Data 
+Protection Regulation (GDPR) law.
+
+A multi-region deployment may have multiple regions within the same geography 
+(a large area like a continent or country), a single region in each of several 
+geographies, or multiple regions in multiple geographies.
+
+Multi-region deployments can exist with a single cloud provider or 
+multiple cloud providers. To learn about multi-cloud deployments, see 
+:ref:`arch-center-paradigms-multi-cloud`.
+
+To learn how to configure multi-region deployments and learn about the
+different types of nodes you can add, see 
+:atlas:`Configure High Availability and Workload Isolation 
+</cluster-config/multi-cloud-distribution/>` in the {+service+}
+documentation.
+
+.. note::
+
+   Multi-region deployments are available only for ``M10`` dedicated
+   {+clusters+} and larger.
+
+Use Cases for Multi-Region Deployments
```

Comment:
This section doesn't flow very well into the next section. The headings are the same but flipped around, and I don't understand how the examples relate to the specific architectures we then describe. Could you mesh these together?

Reply:
LMK if you like this change!